sycl/test-e2e/Basic/built-ins/marray_math.cpp

```diff
@@ -2,9 +2,31 @@
 
 // RUN: %{build} %{mathflags} -o %t.out
 // RUN: %{run} %t.out
+// RUN: %if preview-breaking-changes-supported %{ %{build} %{mathflags} -fpreview-breaking-changes -o %t_preview.out %}
+// RUN: %if preview-breaking-changes-supported %{ %{run} %t_preview.out%}
 
+#include <cmath>
 #include <sycl/sycl.hpp>
 
+template <typename T> T get_ulp_sycl(T x) {
```

Comment:
Could you please add a note saying where the inspiration for these came from?

Reply:
Yes